app/assets/stylesheets/helpers/_reset.scss

```diff
@@ -1,29 +1,115 @@
 // flatten browser default padding and font-sizing so we can cleanly apply our own
 
-/* Borrowed from http://meyerweb.com/eric/tools/css/reset/ */
-div, span,
-h1, h2, h3, h4, h5, h6, p, blockquote, pre,
-a, abbr, acronym, address, big, cite, code,
-del, dfn, em, img, ins, kbd, q, s, samp,
-small, strike, strong, sub, sup, tt, var,
-b, u, i, center,
-dl, dt, dd, ol, ul, li,
-fieldset, form, label, legend,
-table, caption, tbody, tfoot, thead, tr, th, td,
-article, aside, canvas, details, embed,
-figure, figcaption, footer, header, hgroup,
-menu, nav, output, ruby, section, summary,
-time, mark {
+// Borrowed from http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Its odd that collections has its own reset CSS, as we provide something similar through the govuk_template. I wonder what a wraith visual regression test would reveal if the reset was removed.